src/main/java/at/hannibal2/skyhanni/features/fishing/trophy/TrophyFishManager.kt

```diff
@@ -23,6 +26,46 @@ object TrophyFishManager {
     val fishes: MutableMap<String, MutableMap<TrophyRarity, Int>>?
         get() = ProfileStorageData.profileSpecific?.crimsonIsle?.trophyFishes
 
+    private var loadedNeu = false
+
+    @SubscribeEvent
+    fun onNeuPVDataLoaded(event: ProfileDataLoadedEvent) {
```

Comment:
Please add one Layer of abstraction to the ProfileDataLoadedEvent. As we do with all extern events.